docs/Istio-on-karmada-different-network.md

```diff
@@ -0,0 +1,294 @@
+✔ Istio core installed                                                           
+- Processing resources for Istiod.  
+```
+
+### Install Istiod on member1
+
+1、Close Karmada's auto-sync feature when labeling a namespace,because I need to label the same namespace `istio-system`
+in the different clusters differently.
+
+Export `KUBECONFIG` and switch to `karmada host`:
+
+```bash
+export KUBECONFIG=$HOME/.kube/karmada.config
+kubectl config use-context karmada-host
+```
+
+Edit `karmada-controller-manager`deployment
+
+```bash
+kubectl edit deployment karmada-controller-manager -n karmada-system
+```
+
+add `--controllers=-namespace,*` in start command
+
+2、Set the member1 and member2's network
+
+switch to `karmada api-server` and list work about namespace `istio-system`:
+
+```bash
+kubectl config use-context karmada-apiserver
+kubectl get work -A | grep istio-system-
+```
+
+the output will looks like as following:
+
+```bash
+[root@vm1-su-001 istio-1.12.6]# kubectl get work -A | grep istio-system-
+karmada-es-member1   istio-system-f854dc5d9            true     21m
+karmada-es-member2   istio-system-f854dc5d9            true     21m
+```
+
+label namespace `istio-system` on `member1` by using the output above
+
+```bash
+kubectl edit work istio-system-f854dc5d9 -n karmada-es-member1
+```
+
+add `topology.istio.io/network: network1` in `.spec.workload.manifests.metadata.labels`
+
+label namespace `istio-system` on `member2` by using the output above
+
+```bash
+kubectl edit work istio-system-f854dc5d9 -n karmada-es-member2
+```
+
+add `topology.istio.io/network: network2` in `.spec.workload.manifests.metadata.labels`
+
+3、Install istio control plane
+
+Export `KUBECONFIG` and switch to `member1`:
+
+```bash
+export KUBECONFIG="$HOME/.kube/members.config"
+kubectl config use-context member1
+```
+
+```bash
+cat <<EOF | istioctl install -y -f -
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    accessLogFile: /dev/stdout
+  values:
+    global:
+      meshID: mesh1
+      multiCluster:
+        clusterName: member1
+      network: network1
+EOF
+```
+
+4、Install the east-west gateway in `member1`
+
+```bash
+samples/multicluster/gen-eastwest-gateway.sh --mesh mesh1 --cluster member1 --network network1 | istioctl install -y -f -
+```
+
+5、Expose the control plane and service in `member1`
+
+```bash
+kubectl apply -f samples/multicluster/expose-istiod.yaml -n istio-system
+kubectl apply -f samples/multicluster/expose-services.yaml -n istio-system
+```
+
+### Configure `member2` as a remote cluster
+
+1、Enable API ServerAccess to `member2`
+
+switch to `member2`:
+
+```bash
+kubectl config use-context member2
+```
+
+Prepare member2 cluster secret
+
+```bash
+istioctl x create-remote-secret --name=member2 > istio-remote-secret-member2.yaml
+```
+
+Switch to `member1`:
+
+```bash
+kubectl config use-context member1
+```
+
+Apply istio remote secret
+
+```bash
+kubectl apply -f istio-remote-secret-member2.yaml

```

Comment:
Why only apply `secret-member2` in `member1`, but do not apply `secret-member1` in `member2`?

https://istio.io/latest/docs/setup/install/multicluster/multi-primary_multi-network/#enable-endpoint-discovery